Formula/g/gstreamer.rb

```diff
@@ -77,6 +77,7 @@ class Gstreamer < Formula
   depends_on "json-glib"
   depends_on "lame"
   depends_on "libass"
+  depends_on "libnice"
```

Comment:
This has no linkage with `libnice` so it doesn't actually seem to be used. The build is likely just building its own `libnice`, which is incompatible with https://docs.brew.sh/Acceptable-Formulae#stuff-that-requires-vendored-versions-of-homebrew-formulae.